cross/libvpx/Makefile

```diff
@@ -46,6 +46,60 @@ ifeq ($(findstring $(ARCH),$(PPC_ARCHES)),$(ARCH))
 CONFIGURE_ARGS += --target=generic-gnu
 endif
 
+# Removal of GCC additional warning flags required at least for ppc853x-5.2 qoriq-5.2
```

Comment:
Sure, it does. My point here was that instead of adding this block here:
```
ifeq ($(ARCH)-$(TCVERSION), ppc853x-5.2)
+MAKE_ARGS += SPKSRC_GCC_NO_WARN_FLAGS=ON
+endif
[...]
ifeq ($(ARCH)-$(TCVERSION), <all other ppc-based arches)
+MAKE_ARGS += SPKSRC_GCC_NO_WARN_FLAGS=ON
+endif
```
...do this on L45 instead:
```
 ifeq ($(findstring $(ARCH),$(PPC_ARCHES)),$(ARCH))
  CONFIGURE_ARGS += --target=generic-gnu
+MAKE_ARGS += SPKSRC_GCC_NO_WARN_FLAGS=ON
  endif
```